src/profile/components/deleteProfile/DeleteProfile.tsx

```diff
@@ -67,13 +67,21 @@ function DeleteProfile(): React.ReactElement {
     targetId: `delete-profile-button`,
   });
 
+  const [scrollIntoViewRef] = useScrollIntoView(isDeletingProfile);
+
   const [getServiceConnections, { data: serviceConnections }] = useLazyQuery<
     ServiceConnectionsRoot,
     ServiceConnectionsQueryVariables
   >(SERVICE_CONNECTIONS, {
     variables: createServiceConnectionsQueryVariables(i18n.language),
+    fetchPolicy: 'no-cache',
     onCompleted: () => {
+      if (dataLoadState === loadedLoadState) {
+        return;
+      }
       setDataLoadState(loadedLoadState);
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
Sometimes functions cannot be ordered by usage, because they cross-use each other and one usage is always before the definition. 

In this case re-ordering can be done